blacksauce/src/_includes/_header.html

```diff
@@ -0,0 +1,23 @@
+<div class="grid--nav u-black u-marginNone">
+<img class="logo_nav" src="/images/logo.png" alt="blacksauce logo">
+
+<nav class="wrapper site-nav">
+ {# @feedback there is a red = sign but I can't figure out why #}
+		<button id="js-nav-trigger" aria-controls="js-site-nav" aria-expanded="false" class="u-button-reset site-nav__trigger">
+			<span aria-hidden="true" class="menu-icon">
+				 <div class="menu-icon__line"></div>
+				 <div class="menu-icon__line"></div>
+				 <div class="menu-icon__line"></div>
+			</span>
+			<p class="u-brandOrange u-marginNone"> Menu </p> 
+			{#   @feedback the menu keeps hanging off of the nav and I can not seem to position it #}
+		</button>
+		<ul id="js-site-nav" class="u-list-reset site-nav__list" aria-hidden="false" data-visible="false">
+			<li class="site-nav__list-item"><a href="#" class="u-block site-nav__link">Home </a></li>
+			<li class="site-nav__list-item"><a href="#" class="u-block site-nav__link">Catering</a></li>
+			<li class="site-nav__list-item"><a href="#" class="u-block site-nav__link">Biscuit Talk</a></li>
+			<li class="site-nav__list-item"><a href="#" class="u-block site-nav__link">News</a></li>
+			{# @feedback - I am having trouble positioning the links on the desktop view, I tried creating a class for the link and then appling it both to the li and then the a but neither seemed to work. #}
```

Comment:
Do you want them to be in the upper right or some other layout? (The comps look a bit inconsistent, so I'm not 💯 sure.)

If the upper right, you can achieve that by positioning the `.site-nav` class in your grid with `grid-column: 3/4;`

If this isn't what you want, let's chat in class!